internal/runbits/requirements/requirements.go

```diff
@@ -209,6 +222,61 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 		pg = nil
 	}
 
+	if r.Auth.Authenticated() && operation == bpModel.OperationAdded && ns.Type() == model.NamespacePackage {
+		pg = output.StartSpinner(out, locale.Tr("progress_cve_search", requirementName), constants.TerminalAnimationInterval)
+
+		vulnerabilities, err := model.FetchVulnerabilitiesForIngredient(r.Auth, &request.Ingredient{
+			Namespace: ns.String(),
+			Name:      requirementName,
+			Version:   requirementVersion,
+		})
+		if err != nil {
+			return errs.Wrap(err, "Failed to retrieve vulnerabilities")
+		}
+
+		var safe bool
+		if vulnerabilities == nil || (vulnerabilities != nil && vulnerabilities.Vulnerabilities.Length() == 0) {
+			safe = true
+		}
+
+		if !safe {
+			pg.Stop(locale.T("progress_unsafe"))
+			pg = nil
+
+			if r.shouldPromptForSecurity(vulnerabilities) {
+				out.Print("")
```

Comment:
Please don't use `out.Print()` with bare strings, because that causes trouble with structured output. You either want to use `out.Notice()`, or `out.Print(output.Prepare())` with a structure for structured output. I believe this is just extra information for the user, so `output.Notice()` should be sufficient.

Reply:
I usually forget about structure output so this comment was helpful. 

PS. Don't look at the `info` runner :P

This may be something we want to audit and fix in our runners.